Integrations/MISP_V2/MISP_V2.yml

```diff
@@ -19,6 +19,29 @@ configuration:
   name: insecure
   required: false
   type: 8
+- display: Select which event's attribute data fields will be saved to context. Default
```

Comment:
Not clear for how the user will choose “all” maybe mention that by leaving it empty it will take all the fields